src/main/java/io/kestra/plugin/airtable/records/Create.java

```diff
@@ -0,0 +1,263 @@
+                      "Status": "Todo"
+                      "Priority": "High"
+                      "Due Date": "2024-12-31"
+                    typecast: true
+                """
+        ),
+        @Example(
+            title = "Create multiple customer records",
+            full = true,
+            code = """
+                id: create_customers
+                namespace: company.airtable
+
+                tasks:
+                  - id: create_customer_records
+                    type: io.kestra.plugin.airtable.records.Create
+                    baseId: "{{ secret('AIRTABLE_BASE_ID') }}"
+                    tableId: "Customers"
+                    apiKey: "{{ secret('AIRTABLE_API_KEY') }}"
+                    records:
+                      - "Name": "Acme Corp"
+                        "Email": "contact@acme.com"
+                        "Status": "Active"
+                      - "Name": "Beta Inc"
+                        "Email": "hello@beta.com"
+                        "Status": "Prospect"
+                    typecast: true
+                """
+        ),
+        @Example(
+            title = "Create record from dynamic data",
+            full = true,
+            code = """
+                id: create_from_input
+                namespace: company.airtable
+
+                inputs:
+                  - id: customer_name
+                    type: STRING
+                    required: true
+                  - id: product
+                    type: STRING
+                    required: true
+                  - id: amount
+                    type: FLOAT
+                    required: true
+
+                tasks:
+                  - id: create_dynamic_record
+                    type: io.kestra.plugin.airtable.records.Create
+                    baseId: "{{ secret('AIRTABLE_BASE_ID') }}"
+                    tableId: "Orders"
+                    apiKey: "{{ secret('AIRTABLE_API_KEY') }}"
+                    fields:
+                      "Customer": "{{ inputs.customer_name }}"
+                      "Product": "{{ inputs.product }}"
+                      "Amount": "{{ inputs.amount }}"
+                      "Order Date": "{{ now() }}"
+                """
+        )
+    }
+)
+public class Create extends Task implements RunnableTask<Create.Output> {
+
+    private static final int MAX_RECORDS_PER_BATCH = 10;
+
+    @Schema(
+        title = "Airtable base ID",
+        description = "The ID of the Airtable base (starts with 'app')"
+    )
+    @PluginProperty(dynamic = true)
+    @NotNull
+    private Property<String> baseId;
+
+    @Schema(
+        title = "Table ID or name",
+        description = "The ID or name of the table within the base"
+    )
+    @PluginProperty(dynamic = true)
+    @NotNull
+    private Property<String> tableId;
+
+    @Schema(
+        title = "API key",
+        description = "Airtable API key for authentication"
+    )
+    @PluginProperty(dynamic = true)
```

Comment:
You must remove `@PluginProperty` everywhere and use `Property` only for your fields types